scripts/wordpress-check.coffee

```diff
@@ -1,21 +1,28 @@
 parser = require 'parse-rss'
-colors = require('irc-colors')
 
 module.exports = (bot) ->
-
   url = "https://wordpress.org/news/category/releases/feed/"
-  interval = 1000 * 60 * 60 * 2
-  # interval is in ms so 1000 * 60 * 60  is 1 hr
+  interval = 1000 * 60 * 60 * 2     # interval is in ms so 1000 * 60 * 60  is 1 hr
+  published = 0     # 0 = no msg has been sent, 1 = msg has been sent
+
   do wpCheck = ->
     today = new Date
-    parser url, (err,rss)->
-      console.log err if err
-      pubDate = new Date(rss[0].pubDate)
-
-      if pubDate.getMonth() == today.getMonth() && pubDate.getDate() == today.getDate()
-        if 'Security' in rss[0].categories
-          bot.say "#{colors.bold.red('URGENT!')} #{rss[0].title} - #{rss[0].link}\nPublished: #{rss[0].date}", "#ubuntu-si"
-        else
-          bot.say "Sweet #{rss[0].categories.join(" ")} available! #{rss[0].title} - #{rss[0].link}\nPublished: #{rss[0].date}", "#ubuntu-si"
+    sleep = ->
+      parser url, (err,rss)->
+        console.log err if err
+        pubDate = new Date(rss[0].pubDate)
+        if pubDate.getMonth() == today.getMonth() && pubDate.getDate() == today.getDate() && published == 0
+          if 'Security' in rss[0].categories
+            bot.say "URGENT! #{rss[0].title} is available! Read more at  #{rss[0].link}\nPublished: #{rss[0].date}", "#ubuntu-si"
+            published = 1
+          else
+            bot.say "Sweet! #{rss[0].title} is available! Read more at #{rss[0].link}\nPublished: #{rss[0].date}", "#ubuntu-si"
+            published = 1
```

Comment:
set key with timestamp in redis store and  check for it, so that you don't spam chan on each restart/deploy